src/components/PlioQuestion.vue

```diff
@@ -241,6 +271,19 @@ export default {
       });
     },
 
+    updateAndShowProgress(){
+      if (this.plioQuestion.state != "answered"){
+        this.updatedProgressBarInfo[
+          'updatedCompletionPercent'] += ((1/this.updatedProgressBarInfo['totalQuestions'])*100)
+      }
+
+      setTimeout(() => {
+        this.$refs["progressBarRef"].progressTo(
+          this.updatedProgressBarInfo['updatedCompletionPercent']
+        );
+      }, 200)
+    },
+
```

Comment:
2 steps
- calculate the new progress, set it to the `updatedCompletionPercent` key of `updatedProgressBarInfo` object
- wait for 200ms, call the progress bar component's method `progressTo` which basically tells the component to progress to some value. This value is the `updatedCompletionPercent` as calculated in above point

Reply:
Why wait 200 ms here?

Reply:
If I try to find the `this.$refs["progressBarRef"]` instantly without waiting, it has not been injected into the HTML until now (my guess) and it returns `null` but when I wait for some time, it successfully returns the element